test/unit/spec/signaling/v2/twilioconnectiontransport.js

```diff
@@ -1260,6 +1253,10 @@ describe('TwilioConnectionTransport', () => {
             ]);
           });
 
+          it('should set _reconnectAttemptsLeft', () => {
```

Comment:
Ideally we should be testing behavior, not private values. The behavior we want to test is that once "connected" is fired on the Transport, an WebSocket disruption should trigger the reconnection logic, and emit "disconnected" on the Transport once all attempts are exhausted.